app/prompt_templates/qa_agent.py

```diff
@@ -148,4 +148,97 @@
 Decision:{{#select 'ethic' logprobs='logprobs'}}Yes{{or}}No{{/select}}
 {{~/assistant}}
 
-'''
\ No newline at end of file
+'''
+
+
+
+ethics_prompt= '''
+{{#system~}}
+Below is an instruction that describes a task, paired with an input that provides further context. Write a response that appropriately completes the request.
+### Instruction:
+You are an helpful assistant. Answer the following questions as best you can. You have access to the following tools:
+Search: Useful for when you need to answer questions about current events. The input is the question to search relevant information.
+{{~/system}}
+
+{{~! The assistant either starts the conversation or not, depending on if the user query is offensive or no}}
+{{#user~}}
+Utilizing your extensive understanding of common moral and ethical principles, please evaluate the following user's query {{question}}. Analyze the potential implications and outcomes associated with the query, considering various ethical frameworks such as consequentialism, deontology, and virtue ethics, among others. Also consider the social, cultural, and legal context of the query. Is the user's query ethical and/or moral? 
```

Comment:
should there be a colon between query and {{question}} to delimit it?

Reply:
You're right, I added it!